### PR TITLE
Refactor configs

### PR DESCRIPTION
**Pull request by axelsheva**

### CODE REVIEW

File: config/default/constants.ts [hunk 1]
```diff
@@ -1,144 +1,73 @@
...
+    FROZE: {
+        VOTE_MILESTONE: 10, // in seconds
+        REWARDS: {
+            MILESTONES: [
                 10, // 10% For 0-6 months
                 10, // 10% For 7-12 months
                 10, // 8% For 13-18 months
                 10, // 6% For 19-24 months
                 10, // 4% For 25-30 months
                 10  // 2% For 31 months and above
```
Comment (macaqueee) [inline config/default/constants.ts]: always `10`?

File: shared/model/types.ts [hunk 2]
```diff
@@ -7,3 +8,57 @@ export type Filter = {
 
 export type CreateTransactionParams = { trs: TransactionModel<IAsset>, secret: string };
 
+export interface IConstants {
```
Comment (macaqueee) [inline shared/model/types.ts]: `interface` or `type`?

File: shared/config/socket.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import { ServerOptions } from 'socket.io';
+import { DEFAULT_API_SOCKET_PORT, DEFAULT_CORE_SOCKET_PORT, DEFAULT_CORE_RPC_PORT } from 'shared/config';
 
 export const API_HOST = process.env.SERVER_HOST || 'localhost';
```
Comment (macaqueee) [inline shared/config/socket.ts]: `localhost` is DEFAULT_PROPERTY?

File: config/default/constants.ts (hunk 1 above)
Comment (axelsheva) [reply config/default/constants.ts]: This logic not yet fixed

File: shared/config/socket.ts (hunk 3 above)
Comment (axelsheva) [reply shared/config/socket.ts]: This constant never used, I think it must be deleted. And I will add `SERVER_HOST` as a required environment variable in z-schema validation.

File: shared/model/types.ts (hunk 2 above)
Comment (axelsheva) [reply shared/model/types.ts]: `type`

**Pull request merged**